src/Datadog.Trace/SpanContext.cs

```diff
@@ -68,7 +68,7 @@ private SpanContext(ulong? traceId, string serviceName)
         {
             TraceId = traceId > 0
                           ? traceId.Value
-                          : SpanIdGenerator.CreateNew();
+                          : SpanIdGenerator.ThreadInstance.CreateNew();
```

Comment:
Just for convenience, we could have a static `CreateNew` method that internally calls `ThreadInstance.CreateNew`. But I don't have strong feelings about it